continuous_integration/environment.yml

```diff
@@ -0,0 +1,24 @@
+name: cupy
+channels:
+  - defaults
+dependencies:
+  - python=3.11
```

Comment:
Will this conflict if you're building PYTHON_VERSION=3.10? or any non-py311 version?

Reply:
Should work with 3.10.